docker-compose.yml

```diff
@@ -31,8 +31,15 @@ services:
     volumes:
       - ./docsbox:/home/docsbox
       - /media:/home/docsbox/media
+      - /config:/root/.config/libreoffice/4/user
```

Comment:
Do we need some config files on the host machine?

Reply:
We need to add the dir to the volumes to be able to edit the file, it was this issue that made the libre office not assume the changes